monitoring/README.md

```diff
@@ -92,47 +157,108 @@ c) Upload the dashboard JSON files from your local `./grafana` sub-directory int
+   - Use the following commands to obtain the Ip address of the APIServer in the WDS (e.g., `wds1`) and ITS (e.g., `its1-system`) spaces:
+     ```bash
+        kubectl -n wds1-system get pod -l app=kube-apiserver -o jsonpath="{.items[*].status.podIP}"
+     ```
+     ```bash
+        kubectl -n its1-system get pod -l app=vcluster -o jsonpath="{.items[*].status.podIP}"
+     ```
+     
+     If you're using kind clusters then use the following command to obtain the IP address of the host container for your cluster: `$ docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' <docker-container-name>`
+
+     For example: 
+     ```bash
+     docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' cluster1-control-plane
+     ```
+
+     Output:
+     ```bash
+       172.18.0.3
+     ```
+
+Here are the default port number for the APIservers: 
+
+| APIServer Pod | Default Port #|
+| ------------- | ------------- |
+| Host-cluster  |     6443      |
+| WDS space     |     6443      |
+| ITS space     |     8443      |
+| WEC-cluster   |     9444      |
 
-Note: you can also monitor the APIserver of the hosting cluster using the following values: `NS: default` and `APIService: apiserver`
 
 b) API Server Priority & Fairness: input into the following text boxes.
 - `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, *its1-system*, etc.
 - `APIService`: name of the service for a KubeStellar space APIserver, e.g., *wds1*, *wds2*, *vcluster*, etc.
+- `Instance`: hostname and port number for the target KubeStellar space APIserver, e.g., *10.244.0.18:9444*
 
     <img src="images/apiserver-apf-monitoring.png" width="60%" height="80%" title="apiserver-apf-monitoring.png"> 
 
-Note: use the same combinations of `NS` and `APIService` parameters described in `step 5-a` to monitor the APIserver in the WDS space, ITS space or the hosting cluster. 
+**Note**: use the same combinations of `NS` and `APIService` parameters described in `step 5-a` to monitor the APIserver in the WDS space, ITS space, the hosting or WEC clusters. 
 
 
 c) KubeStellar Controllers Monitoring: input into the following text box and drop-down menu. 
 
-- `SpaceNS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, etc.
+- `NS`: name of the KubeStellar space namespace, e.g., *wds1-system*, *wds2-system*, etc.
 
-- `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller* and *status-addon-controller*. 
+- `KSController`: select the KubeStellar controller name. Available options from the drop-down menu: *kubestellar-controller-manager-metrics-service*, *ks-transport-controller*, *status-addon-controller* and *status-agent-controller*. 
+
+- `Instance`: hostname and port number for the target KubeStellar space controller, e.g., *10.244.0.71:8090*
```

Comment:
Do we really need to make the user supply this value? In other words, why include "instance" in the filter? Isn't there always exactly one right "instance" given the other two user inputs?

Reply:
No, that's not the always the case! For example: the apiserver in the hosting cluster & in the WEC cluster have exactly the same namespace (`default`) and job name (`apiserver`) and the only distinction is the `instance` value